backend/src/comment.rs

```diff
@@ -0,0 +1,43 @@
+use crate::database::{
+    models::{Comment, NewComment, SuperUser},
+    Database,
+};
+use rocket::{
+    form::Form,
+    get, post,
+    serde::{json::Json, Serialize},
+};
+
+#[derive(Serialize)]
+pub enum CommentError {
+    CouldNotInsert,
+    CommentNotFound,
+}
+
+#[post("/new", data = "<new_comment>")]
+pub async fn create_comment(
+    new_comment: Form<NewComment>,
+    _user: SuperUser,
```

Comment:
Super user is a CHAOS Application Super User, you probably want to use one of the OrganisationDirector struct methods (or a new one) to create an OrganisationDirector (see https://github.com/csesoc/chaos/blob/main/backend/src/database/models.rs#L109 and https://github.com/csesoc/chaos/blob/main/backend/src/campaigns.rs#L97-L112)